### PR TITLE
Added a new project post

### PR DESCRIPTION
**Pull request by C-NERD**
This is a project i completed in nim, help me review the commit to see if i made it ok. Thanks in advance

### CODE REVIEW

File: jekyll/_posts/2021-02-01-discoverassets.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: "A simple search engine for free game assets"
```
Comment (beef331) [inline jekyll/_posts/2021-02-01-discoverassets.md:2]: This block is supposed to say "January 2021's Month With Nim"

File: jekyll/_posts/2021-02-01-discoverassets.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+author: cnerd
```
Comment (beef331) [inline jekyll/_posts/2021-02-01-discoverassets.md:3]: Should still be `author: The Nim Community

File: jekyll/_posts/2021-02-01-discoverassets.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+---
+
+### discoverAssets
```
Comment (beef331) [inline jekyll/_posts/2021-02-01-discoverassets.md:6]: only `##` for these

File: jekyll/_posts/2021-02-01-discoverassets.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+while working on a game i discovered that being an indie game developer is really difficult partly due to the fact that you are required to create your own game assets
```
Comment (beef331) [inline jekyll/_posts/2021-02-01-discoverassets.md:8]: Capitalization and grammar needs work.
"While", "I", "...difficult, partly..."

**Review by beef331 — COMMENTED** [review]
File should be named `2021-02-01-this-month-with-nim.md`
You should follow this template underneath title box.
```
## YourProjectName

Author: YourName or Nickname here

Your super cool description here maybe even an image, video, or link.
```

**Comment by C-NERD** [comment]
Ok

**Comment by beef331** [comment]
Can just make an issue with the newest iteration on how submissions work.

**Pull request closed without merging**